include/functions/math.hpp

```diff
@@ -182,4 +182,59 @@ BigInt gcd(const long long& num1, const BigInt& num2){
 BigInt gcd(const std::string& num1, const BigInt& num2){
     return gcd(num2, BigInt(num1));
 }
+
+/*    
+    lcm (BigInt, BigInt) :
+    --------------------
+*/
+
+BigInt lcm(const BigInt &num1, const BigInt &num2){
```

Comment:
The `lcm()` function can be written more simply as just this line:
```C++
    return abs(num1 * num2) / gcd(num1, num2);
```

Reply:
Done. Please review once.

Reply:
The single line above is the most efficient as there are no conditions being checked, and has the least number of temporary variables.

Reply:
@faheel 
Ok thanks.
Do I need to do something more in this. I have updated this as asked.